src/utils/getAllReportees.js

```diff
@@ -51,16 +53,11 @@ const findReportees = (completeOrg, email) => {
 };
 
 const getAllReportees = async (userSession, ldapEmail) => {
-  let people;
-  if (process.env.ALTERNATIVE_AUTH) {
-    // if non-LDAP user, get fake data
-    people = await (await fetch('people.json')).json();
-  } else {
-    // LDAP user, retrieve data from the taskcluster secret
-    const secretsClient = userSession.getTaskClusterSecretsClient();
-    const { secret } = await await secretsClient.get(config.taskclusterSecrets.orgData);
-    people = secret.employees;
-  }
+  const peopleGZ = await loadArtifact(userSession, config.artifactRoute, config.peopleTree);
+  const people = JSON.parse(pako.inflate(peopleGZ, { to: 'string' }));
+
+  console.log(people);
```

Comment:
you probably forgot to remove this

Reply:
Oh yeah, and it's far from finished, don't bother reviewing as long as it's in draft mode